public/index.html

```diff
@@ -1,77 +1,90 @@
-<html> 
-    <head>
-        <title> JacobScript </title>
-        <meta name="viewport" content="width=device-width,initial-scale=1.0">
-        <link rel="stylesheet" href="styles.css">
-        <link rel="icon" type="image/x-icon" href="/favicon.ico" />
-        <link href="https://fonts.googleapis.com/css?family=PT+Sans|PT+Sans+Narrow:400,700|PT+Serif" rel="stylesheet">
-    </head>
-    <body>
-        <header>
-            <nav>
-                 <div class = "social-headers">
-                     <a href="#facebook"><img src="images/facebook-header-icon.svg" alt="Facebook Header Icon"></a>
-                     <a href="#twitter"><img src="images/twitter-header-icon.svg" alt="Twitter Header Icon"></a>
-                     <a href="#tumblr"><img src="images/tumblr-header-icon.svg" alt="Tumblr Header Icon"></a>
+<html>
+
+<head>
+    <title>Speak Louder!</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="icon" type="image/x-icon" href="/favicon.ico" />
+    <link href="https://fonts.googleapis.com/css?family=PT+Sans|PT+Sans+Narrow:400,700|PT+Serif" rel="stylesheet" />
+</head>
+
+<body>
+    <header>
+        <nav>
+            <div class="social-headers">
+                <a href="#facebook"><img src="images/facebook-header-icon.svg" alt="Facebook Header Icon"></a>
+                <a href="#twitter"><img src="images/twitter-header-icon.svg" alt="Twitter Header Icon"></a>
+                <a href="#tumblr"><img src="images/tumblr-header-icon.svg" alt="Tumblr Header Icon"></a>
+            </div>
+            <div class="logo">
+                <img src="images/logo.png" alt="Speak Louder logo">
+            </div>
+            <div class="logo-small">
+                <img src="images/logo-small.png" alt="Speak Louder logo small">
+            </div>
+            <div class="menu">
+                <ul>
+                    <li><a href="#archive">Archive</a></li>
+                    <li><a href="#cast">Cast</a></li>
+                    <li><a href="#about">About</a></li>
+                </ul>
+            </div>
+            <div class="burger"><a href="#nav">&equiv;</a></div>
+        </nav>
+    </header>
+    <div id="comic-frame">
+        <div class="panels">
+            <div id="panel-1"><img src="images/panel-1.jpg" alt="Panel 1"></div>
```

Comment:
The main change other than indentation was removing white space around these id attributes.